components/reports/wildfire/ReportFlammabilityMap.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="has-text-centered has-text-weight-bold">
+    <span v-html="title"></span>
+    <div :id="mapID" class="flammability-minimap"></div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.flammability-minimap {
+  height: 15vw;
+  width: 100%;
+}
+</style>
+
+<script>
+import _ from 'lodash'
+import { mapGetters } from 'vuex'
+
+let models = [
+  '5modelAvg',
```

Comment:
I think we generally want this to be `5 Model Average` but I don't think we're labeling this for other charts/graphs, so this is sorta-OK for now.